### PR TITLE
readme: update config and implementation support with CLN

### PR DESCRIPTION
**Pull request by carlaKC**
Updated to include the new json format.

### CODE REVIEW

**Comment by carlaKC** [comment]
@sr-gi or @okjodom could I get 👀  on this? I get v triggered by outdated readmes :(

**Comment by okjodom** [comment]
read through review on mobile.

lgtm

File: README.md [hunk 1]
```diff
@@ -32,37 +38,59 @@ json:
   like to generate.
 
 The example config below sets up the following simulation:
-* Connect to a single node `Alice` to generate activity.
+* Connect to `Alice` running LND to generate activity.
+* Connect to `Bob` running CLN to generate activity.
 * Dispatch 100 msat payments from `Alice` to `Carol` every 10 seconds.
-* Dispatch 100000 msat payments from `Alice` to `Eve` every 5 minutes.
+* Dispatch 100000 msat payments from `Bob` to `Alice` every 5 minutes.
```
Comment (sr-gi) [inline README.md:44]: These two descriptions do not match the provided activity.

`Alice` is actually sending `Carol` 2000 msat every 1 sec
`Bob` is actually sending `Alice` 140000 msat every 50 sec

File: README.md [hunk 2]
```diff
@@ -32,37 +38,59 @@ json:
...
     {
-        // Alice -> Carol
-        "source": "0257956239efc55dd6be91eff40c47749314ccf79cb15f79e30ca12f8622b6de9e",
-        "destination": "02d804ad31429c8cc29e20ec43b4129553eb97623801e534ab5a66cdcd2149dbed",
-        "interval_secs": 10,
-        "amount_msat": 100
+      "CLN": {
+        "id": "0230a16a05c5ca120136b3a770a2adfdad88a68d526e63448a9eef88bddd6a30d8",
+        "address": "https://localhost:10013",
+        "ca_cert": "/path/client.cert",
```
Comment (sr-gi) [inline README.md:61]: This should read `/path/ca.cert`

**Review by sr-gi — COMMENTED** [review]
Post-merge review

**Comment by carlaKC** [comment]
cc: @Emmanuel-Develops @bigmarh, change in config format to allow different types of nodes.

**Comment by Emmanuel-Develops** [comment]
@carlaKC, alright 🫡

so we prompt user to select the type of node then that determines the connection parameters we request from them and then and we create the config in the structure of the readme. This checks out right?

**Pull request merged**